packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -52,6 +66,11 @@ class FlexibleSpaceBar extends StatefulWidget {
   /// Defaults to being adapted to the current [TargetPlatform].
   final bool centerTitle;
 
+  /// Collapse effect while scrolling.
+  ///
+  /// Defaults to Parallax mode.
```

Comment:
rather than "Parallax mode", consider "[CollapseMode.parallax]", so that it links to more information.